gis/JOSM/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                JOSM
+version             16239
+categories          gis editors java
+platforms           darwin
+license             GPL-2+
```

Comment:
Since this is installing a precompiled binary, you should add `supported_archs i386 x86_64` to match the architectures for which that precompiled binary was compiled.

Reply:
Added